viadot/flows/azure_sql_transform_conformed.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any, Dict, List
+# from prefect.triggers import all_successful, any_failed
+from prefect import Flow, task
+
+from ..tasks.azure_sql import  AzureSQLDBQuery
+from ..tasks.prefect_conformed import ReRunFailedFlow
+
+
+query_task = AzureSQLDBQuery()
+prefect_rerun_failed_flow_task = ReRunFailedFlow()
+
+class AzureSQLTransform(Flow): 
```

Comment:
In viadot is the same name of the class. Is it correct? Why here is a separate flow for a rerun? Maybe add `option` to main `AzureSQLTransform` is better idea.